deps/jerry/jerry-core/parser/js/js-lexer.c

```diff
@@ -873,7 +874,13 @@ lexer_parse_number (parser_context_t *context_p) /**< context */
         parser_raise_error (context_p, PARSER_ERR_OCTAL_NUMBER_NOT_ALLOWED);
```

Comment:
In strict mode, only legacyOctalIntegerLiteral is disallowed, but OctalIntegerLiteral is allowed.